docs/release-notes/cloud/cloud-2021-11.md

```diff
@@ -0,0 +1,81 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud November 2021.
+included_jira_versions: ['2021.Q4.4', '2021.Q4.5']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/3.9.48
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/4.0.22
+---
+
+# Cloud November 2021
+
+These release notes are for the Codacy Cloud updates during November 2021.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   Now, Codacy supports linting OpenAPI and AsyncAPI descriptions in either YAML or JSON files using [Spectral](https://stoplight.io/open-source/spectral/){: target="_blank"}. (CY-5088)
+
+-   You can now [use an organization coding standard](../../organizations/using-a-coding-standard.md) to apply the same coding best practices, conventions, or security rules to a group of repositories. (CY-4654)
+
+    ![Organization coding standard](../images/cy-4654.png)
+
+-   The Codacy API now includes [a set of endpoints](https://api.codacy.com/api/api-docs#codacy-api-coding-standards){: target="_blank"} for creating, managing, and applying organization coding standards programmatically. (CY-4617)
+
+## Bug fixes
+
+-   Fixed an issue that prevented the message "Refresh the page to see the results" from being displayed on the commit and pull request pages after a re-analysis was completed. (CY-5187)
```

Comment:
Doesn't need to be a re-analysis, it could've been just a first analysis
```suggestion
-   Fixed an issue that prevented the message "Refresh the page to see the results" from being displayed on the commit and pull request pages after an analysis was completed. (CY-5187)
```